get_query_generators.php

```diff
@@ -287,14 +276,29 @@ function gen_query_list_of_surveys(
  *
  *
  */
-function gen_query_get_survey_data()
+function gen_query_get_survey_data($term)
 {
+    $Specification = "";
+    if($term){
+      $Specification = "AND term = :term";
+    }
     //Get the column of a survey based on the $role
     $survey = "SELECT DISTINCT survey_id, name, default_survey_open as timedate_open, default_survey_close as timedate_close
     FROM surveys
-    WHERE FIND_IN_SET(survey_id,:survey_id);";
+    WHERE FIND_IN_SET(survey_id,:survey_id) $Specification";
+    return $survey;
+}
+/**
+ * Returns sql query that gets list of surveys based on given list of survey_instances
+ */
+function gen_query_get_survey_id()
+{
+    $survey = "SELECT survey_id
```

Comment:
Use string concatination instead of putting newlines in your query. For example

```
$sql = "my really really " .
       "long string with " .
       "lots of parts ";
```